README.md

```diff
@@ -46,3 +46,20 @@ https://alchemycodelab.github.io/web-01-hide-and-seek/
 | :----------------------------------------------------------------------------------- | ----------: |
 | IMPURE: `getRandomHidingSpot() : returns random item from given array` | 2 |
 | IMPURE: `handleGuess(userGuess, correctSpot)` | 2 |
+
+
+
+##Make Plan:
+
+SUGGESTED:
+        1) Call the handleGuess functions first inside event handlers
+        2) Pass the correct arguments to the handleGuess functions
+        3) Finish the handleGuess declaration
+
+MINE: 
```

Comment:
YES, love this personal planning. it will take you so far if you can plan all the pieces of your app.